tests/unit_test/apis/impl/controller_test.py

```diff
@@ -126,7 +126,7 @@ def _setup_system(num_clients=1):
     fl_ctx = mock_server_engine.new_context()
     communicator = WFCommServer()
     controller.set_communicator(communicator)
-    controller.initialize(fl_ctx)
+    controller.initialize_controller(fl_ctx)
```

Comment:
Why change the method name?

Reply:
`initialize()` wasn't originally a method of the controller. This I added in the refactor to replace the responder `initialize_run()`, and think `initialize_controller()` name would be better to differentiate from other `initialize()` methods such as in FLComponentWrapper for example.

Reply:
So there will be name conflicts if we use `initialize()`?

Because I also find ``controller.initialize_controller`` a bit redundant right?

Reply:
Right that does seem redundant, I can change it back to `initialize()` and resolve one minor conflict instead